hyperspy/drawing/utils.py

```diff
@@ -1697,3 +1699,106 @@ def picker_kwargs(value, kwargs=None):
         kwargs['picker'] = value
 
     return kwargs
+
+
+def plot_span_map(sig, spans=1):
```

Comment:
I think commonly `signal` is used instead of abbreviating such a parameter.

Would it be possible to add an option that the different images from the spans are plotted in one (mixed color) figure instead of separate windows? Not sure how hard that is to implement, but would be cool.

Reply:
RE plotting in one image - this is something I wanted to do too.

Sadly I think would be a fair bit of work to sort out.

A while ago I looked into if I could do this using `hyperspy.utils.plot.plot_images`, but as this function essentially uses matplotlib to create the 'overlayed' plot, the plot can't be updated using hyperspy's event system.

Reply:
Fixed in [df32119](https://github.com/hyperspy/hyperspy/pull/3224/commits/df32119f9d5ac98a60b8385a2e7a820b20a9dcb3)